apps/meteor/tests/end-to-end/api/09-rooms.js

```diff
@@ -1847,5 +1847,64 @@ describe('[Rooms]', function () {
 					expect(res.body.room).to.have.property('fname', newDiscussionName);
 				});
 		});
+
+		it('should mark a room as favorite', async () => {
```

Comment:
Can we make the room as default inside this `it` (as well)? I am asking because we need to run the whole `describe` block for this to pass, otherwise, if https://github.com/RocketChat/Rocket.Chat/blob/5d04e9872c7614a3ba60f70b275b836c24e6424b/apps/meteor/tests/end-to-end/api/09-rooms.js#L1767 didn't run the test will fail. I mean, the result will be the same, just to make this independent.

Reply:
idk, since when should an "it" be independent from the describe? I can understand describes being separated, but `it`? 🤔

Reply:
I mean, just to be more explicit this test needs the room as default, so we can have everything inside that it, but that's ok, not a big deal. 👍